docusaurus/src/theme/SearchBar/index.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState, useRef, useCallback, useMemo } from 'react';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import Head from '@docusaurus/Head';
+import { DocSearchButton, useDocSearchKeyboardEvents } from '@docsearch/react';
+import { translate } from '@docusaurus/Translate';
+import { useLocation } from '@docusaurus/router';
+
+import { DocSearchModal } from './DocSearchModal';
+
+import('@docsearch/react/style');
+import('./styles.css');
+
+const baseUrl = '/chat/docs/sdk/';
```

Comment:
Possible to import from `docusaurus.config.js`?

import { baseUrl } from '../../../../docusaurus.config.js'